index.html

```diff
@@ -17,6 +17,9 @@ <h2 class="h2 left-align white">Mozilla Berlin</h2>
         <h2 class="center">Berlin <span class="mozilla--title">moz://a</span> Community Space</h2>
         <p class="center">GSG-Hof Schlesische Straße Gebäude 3, 4th floor<br> Schlesische Straße 27<br> 10997 Berlin</p>
       </div>
+      <h2 class="mozilla--title">Events</h2>
+      <p>We regularly host tech meetups, learning groups and other web-related gatherings, free of charge. Our space is sponsored and maintained by Mozilla, but we operate based on a trust model that enables volunteers to indepently host their own events.</p>
+      <p><strong>Do you want to run an event in the Mozilla Community Space? Please get in touch <a href="https://goo.gl/forms/4fZZzBB9QnCUemyd2">here</a>!</strong></p>
```

Comment:
*Would you like* to run...? (or) *Interested in running*...?

Reply:
I like this very straightforward approach. I would approve this. We can change it at any time.